blog/articles/event-drive-app-knative-eventing-kogito.md

```diff
@@ -0,0 +1,130 @@
+
+Note that the specification allows for both JSON and YAML workflow formats, and we used the YAML one in this example, but one based on JSON is considered equivalent and also parsable by the runtime.
+
+During compilation, the Kogito runtime will parse this YAML file and will generate Java code that represents this workflow definition. The generated code is based on the Quarkus framework. The outcome is an OpenAPI standard REST service that can be deployed anywhere in your architecture.
+
+![Kogito Runtime workflow parse process](images/kogito/kogito-process-workflow-file.png)*Kogito Runtime parsing flow*
+
+In this example, we also added the [Knative Kogito Eventing plugin](https://docs.jboss.org/kogito/release/latest/html_single/#con-knative-eventing_kogito-developing-process-services)  to the project, which means that it can accept Cloud Events through HTTP on the root path, for example:
+
+```bash
+$ curl -X POST \
+      -H "content-type: application/json"  \
+      -H "ce-specversion: 1.0"  \
+      -H "ce-source: /hospital/entry"  \
+      -H "ce-type: new.patients.events"  \
+      -H "ce-id: 12346"  \
+      -d "{ \"name\": \"Mick\", \"dateOfBirth\": \"1983-08-15\", \"symptoms\":[\"seizures\"]}" \
+http://localhost:8080
+```
+
+When sending this request to the service, it will start a new instance of the workflow and perform all the operations defined by it.
+
+There's much more included in this example. For comprehensive instructions on how to deploy and build it, please see the example page on the [Github repository](https://github.com/kiegroup/kogito-examples/tree/master/serverless-workflow-functions-events-quarkus).
+
+### Knative Eventing Integration
+
+Based on this generated service, [an image can be built](https://docs.jboss.org/kogito/release/latest/html_single/#proc-kogito-deploying-on-kubernetes_kogito-deploying-on-openshift) to be deployed with the [Kogito Operator](https://docs.jboss.org/kogito/release/latest/html_single/#con-kogito-operator-and-cli_kogito-deploying-on-openshift) on a Kubernetes cluster with [Knative Eventing installed](https://knative.dev/docs/eventing/). The operator will create all the necessary Knative resources to have this service configured and listening to a [Broker](https://knative.dev/docs/eventing/broker/):
+
+![Knative and Kogito integration](images/kogito/knative-eventing-kogito-operator.png)*Knative Eventing and Kogito Operator integration*
+
+The Kogito Operator will create a [Knative Trigger](https://knative.dev/docs/eventing/triggers/) resource that will link the service and the Broker together. In this example, it will filter events of type `new.patients.events`. Meaning that every time a new event of this type comes to the Broker, it will be redirected to the Kogito service.
```

Comment:
```suggestion
The Kogito Operator creates a Knative [Trigger](https://knative.dev/docs/eventing/triggers/) resource, that links the service and the broker together. In this example, it will filter events of type `new.patients.events`. This means that every time a new event of this type comes to the broker, it will be redirected to the Kogito service.
```